### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by plentylife** [sdelements/lets-chat#781]
There haven't been any commits in months. Is this project alive?

**Comment by plentylife** [comment]
What is it that you've been working on, if you don't mind me asking?

**Comment by hhaidar** [comment]
Haha I'm not sure why you'd be interested, but mainly a react rewrite of SD Elements (our main app) and now I'm working on a new product with a new team.

**Comment by plentylife** [comment]
Curiosity killed the cat :P

**Comment by RoestVrijStaal** [comment]
@hhaidar Which rival of Let's chat would you recommend?

**Comment by hhaidar** [comment]
@RoestVrijStaal probably @mattermost or RocketChat. I can't really comment though, because I haven't ran them apart from demos.

If it helps, our company ended up switching to @slackhq last January.

**Comment by plentylife** [comment]
Mattermost is open source, slack is not.

**Comment by RailRunner166** [comment]
@hhaidar i love lets chat, because of its light footprint on my server and easy manageability, so i have forked it and i cant wait to see where it goes, i will continue with implementing markdown support first then move on to features like direct messages etc, if you all want to follow it, you can see my fork 😃

**Comment by strypey** [comment]
Hi @LizAinslie , are you still maintaining this fork? FYI I curate a list of [potential free code replacements for Slack](https://wiki.p2pfoundation.net/Slack_Alternatives). It's worth considering software that implements a federation protocol like XMPP (Snikket) or matrix (Element or Gitter), so you don't end up stuck in yet another chat silo.

**Comment by LizAinslie** [comment]
@strypey unfortunately I am not. My company, Synapse Technologies, will provide its own chat solution but it will also be closed-source. While Matrix and XMPP protocol support are in consideration and for the most part planned, on-premises licensing will be priced according to enterprise service as per the business model we operate on. Another thing to keep in mind is that this is all in early stages, and enterprise licensing won't be made available until November 2021. I guess I ought to plug the Synapse website here for anyone who may be interested, even though there's nothing but an NGINX default page there as of yet. (I'll get the landing transferred from our old host soon, but for now enjoy NGINX: https://synapsetech.dev)

And while the paid licensing isn't exactly what a lot of people want to hear, all Synapse services (including our chat service) will be free (/freemium) for individual users to benefit from, so it's not entirely locked down to enterprise plans.

And it goes without saying that we may also shift our plans towards open-source community releases alongside the full-fledged option.

**TL;DR is no, I am not still maintaining my fork, but I will have an alternative to offer in the very near future.**

Sorry for the (absurdly) late response by the way, I've been out for a while for personal reasons. Hope I helped though!
